modules/demo/src/main/webapp/index.html

```diff
@@ -13,6 +13,7 @@
 
 <body>
   <script type="module" src="/main.jsx"></script>
+  <!--script type="module" src="/aladin.jsx"></script-->
```

Comment:
commented

Reply:
Maybe another comment here to explain that?

Reply:
what is a comment?